src/data-sources/DataSourceList.tsx

```diff
@@ -99,15 +100,24 @@ const DataSourceList = () => {
 						{ [
 							{
 								icon: AirtableIcon,
-								label: __( 'Airtable', 'remote-data-blocks' ),
+								label: __( SUPPORTED_SERVICES_LABELS.airtable, 'remote-data-blocks' ),
```

Comment:
After these changes we don't need `__()` function.

Reply:
We'll still need the `__()` to translate the label value, won't we ?

Reply:
As far as I know `__()` doesn't work on variables without using `sprintf`. We should use `__()` in `SUPPORTED_SERVICES_LABELS` to translate these strings on source.

Reply:
95993fa